### PR TITLE
Update Design System team page

### PR DESCRIPTION
**Pull request by kellylee-gds**
- added Anika
- updated Charlotte's job title
- restructured content to remove squads and update team leads list

### CODE REVIEW

File: src/design-system-team.md [hunk 1]
```diff
@@ -12,28 +12,29 @@ The GOV.UK Design System at the [Government Digital Service](https://www.gov.uk/
 
 If you want to contact the team you can [get in touch via email or Slack](/get-in-touch/).
 
-- Izabela Podralska – Delivery Manager
+### Team leads
+
+- Chris Ballatine-Thomas – Senior Interaction Designer
```
Comment (owenatgov) [inline src/design-system-team.md:17]: Teeny tiny typo
```suggestion
- Chris Ballantine-Thomas – Senior Interaction Designer
```

Comment (kellylee-gds) [reply src/design-system-team.md:17]: Thank you!

**Review by owenatgov — APPROVED** [review]

**Comment by kellylee-gds** [comment]
@owenatgov are you bovved about the squashing the commits?

**Comment by owenatgov** [comment]
@kellylee-gds It's not the end of the world but if you're up for squashing it then I'd appreciate it 👍🏻

**Comment by kellylee-gds** [comment]
@owenatgov I still don't know how to do it 🤦🏼‍♀️

**Comment by owenatgov** [comment]
@kellylee-gds No worries! Done it for ya. Merge at will.

**Pull request merged**